src/main/java/frc/trigon/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,137 @@
+import edu.wpi.first.wpilibj2.command.sysid.SysIdRoutine;
+import frc.trigon.robot.subsystems.MotorSubsystem;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXMotor;
+import org.trigon.hardware.phoenix6.talonfx.TalonFXSignal;
+import org.trigon.utilities.Conversions;
+
+public class Climber extends MotorSubsystem {
+    private final TalonFXMotor
+            rightMotor = ClimberConstants.RIGHT_MOTOR,
+            leftMotor = ClimberConstants.LEFT_MOTOR;
+    private final DynamicMotionMagicVoltage
+            groundedPositionRequest = new DynamicMotionMagicVoltage(
+            0,
+            ClimberConstants.MAX_GROUNDED_VELOCITY,
+            ClimberConstants.MAX_GROUNDED_ACCELERATION,
+            0
+    ).withSlot(ClimberConstants.GROUNDED_SLOT).withEnableFOC(ClimberConstants.ENABLE_FOC),
+            onChainPositionRequest = new DynamicMotionMagicVoltage(
+                    0,
+                    ClimberConstants.MAX_ON_CHAIN_VELOCITY,
+                    ClimberConstants.MAX_ON_CHAIN_ACCELERATION,
+                    0
+            ).withSlot(ClimberConstants.ON_CHAIN_SLOT).withEnableFOC(ClimberConstants.ENABLE_FOC);
+    private final VoltageOut voltageRequest = new VoltageOut(0).withEnableFOC(ClimberConstants.ENABLE_FOC);
+    private ClimberConstants.ClimberState targetState = ClimberConstants.ClimberState.RESTING;
+
+    public Climber() {
+        setName("Climber");
+    }
+
+    @Override
+    public void periodic() {
+        rightMotor.update();
+        leftMotor.update();
+        updateMechanisms();
+    }
+
+    @Override
+    public void drive(Measure<Voltage> voltageMeasure) {
+        driveRightMotor(voltageMeasure.plus(Units.Volts.of(calculateStayingInPlaceFeedback(rightMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight))));
+        driveLeftMotor(voltageMeasure.plus(Units.Volts.of(calculateStayingInPlaceFeedback(leftMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight))));
+    }
+
+    @Override
+    public void updateLog(SysIdRoutineLog log) {
+        log.motor("RightClimberMotor")
+                .linearPosition(Units.Meters.of(rightMotor.getSignal(TalonFXSignal.POSITION)))
+                .linearVelocity(Units.MetersPerSecond.of(rightMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(rightMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE) - calculateStayingInPlaceFeedback(rightMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight)));
+        log.motor("LeftClimberMotor")
+                .linearPosition(Units.Meters.of(leftMotor.getSignal(TalonFXSignal.POSITION)))
+                .linearVelocity(Units.MetersPerSecond.of(leftMotor.getSignal(TalonFXSignal.VELOCITY)))
+                .voltage(Units.Volts.of(leftMotor.getSignal(TalonFXSignal.MOTOR_VOLTAGE) - calculateStayingInPlaceFeedback(leftMotor.getSignal(TalonFXSignal.POSITION), targetState.affectedByRobotWeight)));
```

Comment:
Instead of inputting `targetState.affectedByRobotWeight` to the sysid, have a constant for whether sysid is grounded